apptools/scripting/recorder.py

```diff
@@ -1,9 +1,6 @@
 """
 Code to support recording to a readable and executable Python script.
 
-TODO:
-    - Support for dictionaries?
```

Comment:
it'll be better to replace the TODO, which makes sphinx freak out, with a FIXME/NOTE and keep the comment about supporting dictionaries.